tests/rbd/rbd_utils.py

```diff
@@ -396,10 +396,27 @@ def image_info(self, pool_name, image_name):
         try:
             image_info = json.loads(out)
         except TypeError:
-            log.error("Failed to recieve json complaint image info")
+            log.error("Failed to receive json complaint image info")
             return 1
         return image_info
 
+    def disable_rbd_feature(self, pool_name, image_name, feature_name):
```

Comment:
we already have image_feature_disable() method in rbd_mirror_util.py, we can utilize that instead of creating new one for as going forward we are combining both the utils in one.

Reply:
As per the discussion, we have decided to create methods in rbd utils and not use the rbd mirror methods for this